### ISSUE TITLE
error: ‘content_length’ may be used uninitialized

### ISSUE THREAD

**Issue by bagder** [rustls/rustls-ffi#124]
rustls-ffi 0.7.0 fails to build in [our CI](https://curl.zuul.vexxhost.dev/build/dc62a0b548f94bd680f36837db53aaf0) due to this error. The code seems to look like this in master as well.
~~~
cbindgen --lang C > src/crustls.h
WARN: Skip crustls::RUSTLS_CRATE_VERSION - (not `pub`).
WARN: Missing `[defines]` entry for `unix` in cbindgen config.
WARN: Missing `[defines]` entry for `unix` in cbindgen config.
cc -o target/client.o -c tests/client.c -Werror -Wall -Wextra -Wpedantic -g -I src/ -O3
tests/client.c: In function ‘send_request_and_read_response’:
tests/client.c:319:42: error: ‘content_length’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
            conn->data.len >= headers_len + content_length) {
                              ~~~~~~~~~~~~^~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
Makefile:40: recipe for target 'target/client.o' failed
make: *** [target/client.o] Error 1

**Comment by bagder** [comment]
It appear to be a false positive, but perhaps we could still clear the variable at declaration to avoid it?

**Comment by bagder** [comment]
Thanks, fixed via #125
